themes/godotengine/layouts/home.htm

```diff
@@ -143,10 +166,10 @@
   <div class="container flex eqsize responsive">
 
     <div class="padded">
-      <h2>The game engine you waited&nbsp;for.</h2>
+      <h2>The game engine you've been waiting for.</h2>
```

Comment:
See https://github.com/godotengine/godot-website/pull/381, there is a concern about how it would break into lines at different sizes.

Reply:
🤔 IMO awkward line breaks are better than an slightly awkward phrasing as the very first text on the homepage, but I can remove for now

Reply:
Okay fixed this so it no longer haves any widows